packages/smooth_app/assets/onboarding/sample_product_data.json

```diff
@@ -0,0 +1 @@
fat":"low","salt":"low","saturated-fat":"low","sugars":"low"},"nutriments":{"carbohydrates":1.4,"carbohydrates_100g":1.4,"carbohydrates_serving":2.18,"carbohydrates_unit":"g","carbohydrates_value":1.4,"energy":76,"energy-kcal":20,"energy-kcal_100g":20,"energy-kcal_serving":31.2,"energy-kcal_unit":"kcal","energy-kcal_value":20,"energy-kj":76,"energy-kj_100g":76,"energy-kj_serving":119,"energy-kj_unit":"kj","energy-kj_value":76,"energy_100g":76,"energy_serving":119,"energy_unit":"kj","energy_value":76,"fat":0.3,"fat_100g":0.3,"fat_serving":0.468,"fat_unit":"g","fat_value":0.3,"fiber":2.8,"fiber_100g":2.8,"fiber_serving":4.37,"fiber_unit":"g","fiber_value":2.8,"fruits-vegetables-nuts-estimate-from-ingredients_100g":20,"nova-group":3,"nova-group_100g":3,"nova-group_serving":3,"nutrition-score-fr":-8,"nutrition-score-fr_100g":-8,"proteins":1.8,"proteins_100g":1.8,"proteins_serving":2.81,"proteins_unit":"g","proteins_value":1.8,"salt":0.01,"salt_100g":0.01,"salt_serving":0.0156,"salt_unit":"g","salt_value":0.01,"saturated-fat":0.1,"saturated-fat_100g":0.1,"saturated-fat_serving":0.156,"saturated-fat_unit":"g","saturated-fat_value":0.1,"sodium":0.004,"sodium_100g":0.004,"sodium_serving":0.00624,"sodium_unit":"g","sodium_value":0.004,"sugars":0.1,"sugars_100g":0.1,"sugars_serving":0.156,"sugars_unit":"g","sugars_value":0.1},"nutrition_grade_fr":"a","product_name":"Sliced mushrooms in water","product_quantity":"156","quantity":"156g (290g)","selected_images":{"front":{"display":{"en":"https://images.openfoodfacts.org/images/products/20462123/front_en.27.400.jpg"},"small":{"en":"https://images.openfoodfacts.org/images/products/20462123/front_en.27.200.jpg"},"thumb":{"en":"https://images.openfoodfacts.org/images/products/20462123/front_en.27.100.jpg"}},"ingredients":{"display":{"en":"https://images.openfoodfacts.org/images/products/20462123/ingredients_en.14.400.jpg"},"small":{"en":"https://images.openfoodfacts.org/images/products/20462123/ingredients_en.14.200.jpg"},"thumb":{"en":"https://images.openfoodfacts.org/images/products/20462123/ingredients_en.14.100.jpg"}},"nutrition":{"display":{"en":"https://images.openfoodfacts.org/images/products/20462123/nutrition_en.29.400.jpg"},"small":{"en":"https://images.openfoodfacts.org/images/products/20462123/nutrition_en.29.200.jpg"},"thumb":{"en":"https://images.openfoodfacts.org/images/products/20462123/nutrition_en.29.100.jpg"}}},"serving_size":"156g","states_tags":["en:to-be-completed","en:nutrition-facts-completed","en:ingredients-completed","en:expiration-date-completed","en:packaging-code-to-be-completed","en:characteristics-to-be-completed","en:origins-to-be-completed","en:categories-completed","en:brands-completed","en:packaging-completed","en:quantity-completed","en:product-name-completed","en:photos-to-be-validated","en:packaging-photo-to-be-selected","en:nutrition-photo-selected","en:ingredients-photo-selected","en:front-photo-selected","en:photos-uploaded"]},"status":1,"status_verbose":"product found"}
```

Comment:
Looks like this is currently a whole english product, later on we should definitely have one for each language. @teolemon  @stephanegigandet are there already any showcase products for at least a few languages and if not, is there a way to search for products with the most data per language.